src/context/volume-context.js

```diff
@@ -1,17 +1,21 @@
 import { createContext, useContext, useReducer } from "react";
+import config from "../assets/config.json";
 
 const VolumeContext = createContext();
-const pathPrefix = "./assets/models/";
 
 // Custom component to provide the Volume context
 function VolumeProvider(props) {
   const [state, dispatch] = useReducer(volumeReducer, {
-    options: {
-      measurement: "salt",
-      season: "summer",
-      tide: "low",
+    selection: {
+      season: config.season[0],
+      tide: config.tide[0],
+      measurement: config.measurement[0],
     },
-    path: `${pathPrefix}summer_low_salt.png`,
+    slices: 55,
+    extension: ".png",
+    x_spacing: 2.0,
+    y_spacing: 2.0,
+    z_spacing: 1.0,
   });
```

Comment:
```suggestion
    slices: config.slices,
    extension: config.extension,
    x_spacing: config.x_spacing,
    y_spacing: config.y_spacing,
    z_spacing: config.z_spacing,
  });
```

Reply:
instead of this approach, you could destructure config on import, then use those values directly (I think, probably, but also maybe not and if not you could destructure it before initializing state)

so
```js
import { season, measurement, tide, extension, slices, x_spacing, y_spacing, z_spacing } from "../assets/config.json";
```

then you could just pass those variables without the `: config.blah` in the state initialization

Reply:
Oh that's true too. I'm going to open another pr with some other context changes, I'll swap things there?